index.js

```diff
@@ -24,6 +25,10 @@ function UTError(x) {
             }.bind(this));
         }
     }
+    if (!this.code) {
+        // unknown error in iso8583
+        this.code = 96;

```

Comment:
Is there a way to not put iso8583 specifics in this module?

Reply:
Depends on whether anyone suggests a better nomenclature.